code/modules/events/shuttle_loan.dm

```diff
@@ -99,6 +104,9 @@
 			priority_announce("Cargo: A neighboring station needs some help handling some paperwork. Could you help process it for us?", "CentCom Paperwork Division")
 			thanks_msg = "The cargo shuttle should return in five minutes. Payment will be rendered when the paperwork is processed and returned."
 			bonus_points = 0 //Payout is made when the stamped papers are returned
+		else
+			log_game("Shuttle Loan event could not find [dispatch_type] event to offer.")
+			kill()
```

Comment:
Since you're killing the event, maybe line 78 (``SSshuttle.shuttle_loan = src``) should take place after this so it doesn't break anything?

Reply:
@Rhials You'll have to early return here too because otherwise you're setting shuttle_loan to something that no longer exists